.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -27,10 +27,11 @@ KFP SDK version: <!-- Please attach the output of this shell command: $pip list
 
 /kind bug
 <!-- Please include labels by uncommenting them to help us better triage issues, choose from the following -->
-<!--
-// /area frontend
-// /area backend
-// /area sdk
-// /area testing
-// /area engprod
--->
+
+<!-- /area frontend -->
+<!-- /area backend -->
+<!-- /area sdk -->
+<!-- /area testing -->
+<!-- /area engprod -->
```

Comment:
These are probably what users wouldn't choose for bugs

Reply:
I removed engprod because users won't know what it is. For others, I think user and community contributor will have some general idea about them. This is optional, so they can leave it blank if they don't know the area.